### PR TITLE
Add c-blosc2 and python-blosc2 as subdirectories of the website

### PR DESCRIPTION
**Pull request by datapythonista**
When building the website, this will also fetch and build the docs of c-blosc2 and python-blosc2 and will them making available as https://blosc.org/c-blosc2/ and https://blosc.org/python-blosc2/

You can download a zip with the resulting website, downloading the `blosc-website` artifact in https://github.com/datapythonista/blogsite/actions/runs/2884599102

There are just few things I'd do before moving this to production:
- Set up the job so it's triggered for commits or tags in the c-blosc2 and python-blosc2 repos, or set it up to run nightly
- This is now building the development (`master` branch) docs, if we want the latest release version, we should call something like `git checkout $(git describe --tags $(git rev-list --tags --max-count=1))` in the fetched git repos
- So far it's just creating the artifact, we need to upload to GitHub pages when everything is ready
- I'd update in the sphinx projects of the other repos to use the same theme in both, and edit the main navigation bar so the links are the same as in the website. This will make the navigation among websites more natural

Also, if you can have a look at the names here. Maybe it makes more sense to use `Docs C` and `Docs Python` instead of `c-blosc2` and `python-blosc2`. And same for the paths. It's trivial to change, but I'm not sure what makes more sense.

CC: @FrancescAlted

### CODE REVIEW

**Comment by FrancescAlted** [comment]
Ups, I am afraid that this felt into the cracks; sorry for the delay.

That looks good to me.  Regarding your questions, I'm not a big fan of consuming CPU cycles unnecessarily, so I'd prefer to update the docs when we have: 1) a tag in c-blosc2 repo (meaning a new release), 2) a tag in python-blosc2 repo (meaning a new release) and 3) a commit in this repo (meaning that we have a new blog that we want to publish).

> Also, if you can have a look at the names here. Maybe it makes more sense to use `Docs C` and `Docs Python` instead of `c-blosc2` and `python-blosc2`. And same for the paths. It's trivial to change, but I'm not sure what makes more sense.

I actually like the current name schema, where users can easily see the match between project names (C-Blosc2 and Python-Blosc2; please respect the caps here) and the corresponding docs section.

Thank you!

**Comment by datapythonista** [comment]
This should be now ready.

blosc_website

There are some changes pending, but not to this repo but the others. What I'd do:
- Trigger this job from the other repos on a tag push as suggested (that'll be a github action in the other repos for pushing tag event, that will do an http request that will trigger this one to run)
- Unify the themes of the docs of the 3 projects
- Update the navbar so all the docs have the same items
- Use the same conventions (directory name, requirements file name...) for the docs in the 3 repos (unless there is a reason not too). This will simplify this a bit

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+    - name: Install Python-Blosc  # Python-Blosc Sphinx conf.py imports blosc, not sure why
+      run: pip install -e $(pwd)/python-blosc/
+    - name: Install dependencies Python-Blosc2
+      run: pip install -r python-blosc2/requirements-doc.txt
+    - name: Pin pygments # Sphinx fails without this: 'HtmlFormatter' object has no attribute 'get_linenos_style_defs'
+      run: pip install pygments==2.11.2
+
+    # Build websites
+    - name: Build Nikola website
       uses: getnikola/nikola-action@v3
```
Comment (FrancescAlted) [inline .github/workflows/main.yml]: Following [this](https://github.com/getnikola/nikola-action), should be using `getnikola/nikola-action@v4` or `getnikola/nikola-action@v5` ?

**Review by FrancescAlted — COMMENTED** [review]
Besides my comment, GH actions is raising an error currently.  Should we be worried about this?

**Comment by datapythonista** [comment]
I guess it makes sense to use the latest version of the `nikola-action` we use, updated.

The error in actions is not for this version of the PR, is from a previous version. This action is currently only running on commits to master, so it didn't run except some tests I did where I forced it to run, to see if anything was failing. It should work now, but can't tell for sure until this is merged. If it fails, no big deal, just the website won't be updated until we fix it.

**Review by FrancescAlted — APPROVED** [review]

**Comment by FrancescAlted** [comment]
Thank you!

**Comment by FrancescAlted** [comment]
Hmm, this is still crashing.  Marc, I have invited your as admin.  Feel free to add any token you might need. Thanks!

**Pull request merged**